storybook/style.scss

```diff
@@ -1,14 +1,11 @@
-/**
- * External dependencies
- */
-@import '@wordpress/components/build-style/style';
-
 /**
  * Internal Dependencies
  */
+@import '../client/stylesheets/index';
```

Comment:
there is some style pollution from our main stylesheet where it is the only reason some components display properly. This is tech debt that we need to fix at some point.

Reply:
This may be because this file included Gutenberg component styles: `@import './shared/gutenberg-components.scss';`. This could go away if we stop bundling wp components.